src/app/user-manager/services/user.service.ts

```diff
@@ -12,7 +12,7 @@ const DB_PATH = '/users';
 export class UserService {
 
   private _users: BehaviorSubject<User[]>;
-  private internalUserSubscription: User[];
+  internalUserSubscription: User[];
```

Comment:
This will be changed back to a private variable but I am currently testing with it.

Reply:
Yep, makes sense to me